lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -314,6 +314,12 @@ export class StandardController implements IController {
             return;
         }
 
+        if(!this.isBrowserEnvironment) {
```

Comment:
Should we also log that we are in non-browser environment?

Reply:
Good idea! Done.